src/index.js

```diff
@@ -97,36 +91,29 @@ export default class ImageTool {
      * Tool's initial config
      */
     this.config = {
-      endpoints: config.endpoints || '',
```

Comment:
You need to add it back because endpoints are not passed to the `Uploader` constructor

Reply:
https://github.com/editor-js/image/pull/20/files#diff-64fd4267c6e48b0ebfe33be416b27348R3

Reply:
> You need to add it back because endpoints are not passed to the `Uploader` constructor

Thx, fix it.